managed/CustomGroup_Currency_Exchange_rates.mgd.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+return [
+  [
+    'name' => 'CustomGroup_Currency_Exchange_rates',
+    'entity' => 'CustomGroup',
+    'cleanup' => 'unused',
+    'update' => 'always',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'currency_exchange_rates',
+        'title' => 'Currency Exchange rates',
+        'extends' => 'Contribution',
+        'extends_entity_column_value' => NULL,
+        'style' => 'Inline',
+        'collapse_display' => TRUE,
+        'help_pre' => '<p data-renderer-start-pos="16722">Stores the sales tax currency conversion data for display on invoice.</p>',
+        'help_post' => '',
+        'weight' => 34,
+        'is_active' => TRUE,
+        'is_multiple' => FALSE,
+        'min_multiple' => NULL,
+        'max_multiple' => NULL,
+        'collapse_adv_display' => TRUE,
+        'is_reserved' => FALSE,
+        'is_public' => FALSE,
+        'icon' => '',
+        'extends_entity_column_id' => NULL,
+      ],
+    ],
+  ],
+  [
+    'name' => 'CustomGroup_Currency_Exchange_rates_CustomField_Sales_Tax_Currency',
+    'entity' => 'CustomField',
+    'cleanup' => 'unused',
+    'update' => 'always',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'custom_group_id.name' => 'Currency_Exchange_rates',
+        'name' => 'sales_tax_currency',
+        'label' => 'Sales Tax Currency',
+        'data_type' => 'String',
+        'html_type' => 'Select',
+        'default_value' => NULL,
+        'is_required' => FALSE,
+        'is_searchable' => FALSE,
+        'is_search_range' => FALSE,
+        'help_pre' => NULL,
+        'help_post' => NULL,
+        'mask' => NULL,
+        'attributes' => NULL,
+        'javascript' => NULL,
+        'is_active' => TRUE,
+        'is_view' => FALSE,
+        'options_per_line' => NULL,
+        'text_length' => 255,
+        'start_date_years' => NULL,
+        'end_date_years' => NULL,
+        'date_format' => NULL,
+        'time_format' => NULL,
+        'note_columns' => 60,
+        'note_rows' => 4,
+        'column_name' => 'sales_tax_currency',
+        'serialize' => 0,
+        'filter' => NULL,
+        'in_selector' => FALSE,
+        'option_group_id.name' => 'currencies_enabled',
+      ],
+    ],
+  ],
+  [
+    'name' => 'CustomGroup_Currency_Exchange_rates_CustomField_Rate_1_Unit_Tax_Currency',
+    'entity' => 'CustomField',
+    'cleanup' => 'unused',
+    'update' => 'always',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'custom_group_id.name' => 'Currency_Exchange_rates',
```

Comment:
you changed the name above to be small letter as per my comment, so you need to update it here too.